com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java

```diff
@@ -178,6 +188,37 @@ public void clearState(ThreadReference thread) {
         }
     }
 
+    private boolean ensureDebugTarget(VirtualMachine vm) {
+        String projectName = (String) options.get(Constants.PROJECTNAME);
```

Comment:
If debugTarget is not null, this line is not necessary ?

Reply:
sure